README.md

```diff
@@ -37,11 +37,15 @@ source 'https://github.com/salemove/glia-ios-podspecs.git'
 user_frameworks!
 
 target 'YOUR_TARGET_NAME' do
-    pod 'GliaWidgets', git: 'https://github.com/salemove/ios-sdk-widgets', tag: 'SDK_VERSION'
+    pod 'GliaWidgets
 end
 
 ```
 
+## Swift Package Manager
+
+For integration, the `GliaWidgets` into you project via [Swift Package Manager](https://www.swift.org/package-manager/) add dependencies `https://github.com/salemove/ios-sdk-widgets`.
```

Comment:
I don't understand what is trying to be said here. As this is public maybe we should involve product writers.

Reply:
I'll request techwriters to review README.md file in general, because it has been created a pretty old time ago and I'm not sure it has been validated.